docs/CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   using the installer script that runs it on the Docker Engine infrastructure.
   ([#308](https://github.com/TheRacetrack/racetrack/issues/308))
 
+- Command `racetrack get auth-token` prints out current auth token.
+  It can be used in CLI scripts: `curl -H "X-Racetrack-Auth: $(racetrack get auth-token)"`
+  ([#357](https://github.com/TheRacetrack/racetrack/issues/357))
+- Command `racetrack login --username ADMIN` allows you to log in with your username and password and saves the auth token without having to visit the Dashboard page.
```

Comment:
I assume it's like `racetrack --<username> <password>`? It's not entirely clear to me. I've never seen a CLI login function like this, I think `racetrack --username <username> --password <password>` is less surprising. 

I also think that it probably shouldn't say ADMIN here, regardless of whether you agree with the above or not.